addons/account/models/account_invoiceable_line.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models
+from odoo.tools import format_date
+
+
+class InvoiceableLine(models.AbstractModel):
+    _name = 'account.invoiceable.line'
+    _description = 'Enables downpayment support for child records of account.invoiceable.order that can be down payments'
+
+    is_downpayment = fields.Boolean(string="Is a down payment",
+                                    help="Down payments are made when creating invoices from an order."
+                                         " They are not copied when duplicating an order.")
+
+    # To override
+    invoice_lines = fields.One2many('account.move.line')
```

Comment:
Changing this would have very far-reaching consequences throughout multiple modules across accounting, sales and logistics team scopes. So my counter-proposal would maybe be to do this renaming in a separate refactor PR instead of this one?